packages/core/content-type-builder/admin/src/icons/Curve.js

```diff
@@ -1,26 +1,53 @@
-import React from 'react';
+import React, { memo } from 'react';
```

Comment:
Why do you need the memo?

Reply:
Also it appears that this component does not work on Safari:

<img width="528" alt="Screenshot 2021-09-14 at 05 45 17" src="https://user-images.githubusercontent.com/13311463/133191743-c5eb8ef7-4cee-4f6a-8698-c5a2909687ee.png">


However it's perfect on both Chrome and Safari well done!

Reply:
ah, legacy code